src/natcap/invest/ui/usage.py

```diff
@@ -115,12 +115,15 @@ def _merge_bounding_boxes(bb1, bb2, mode):
         # Using gdal.OpenEx to check if an input is spatial caused the
         # model to hang sometimes (possible race condition), so only
         # get the bounding box of inputs that are known to be spatial.
+        # Also eliminate any string paths that are empty to prevent an
+        # exception.
         spatial_info = None
-        if args_spec['args'][key]['type'] == 'raster':
+        if args_spec['args'][key]['type'] == 'raster' and value.strip() != '':
```

Comment:
I guess at this point we've passed validation so we know that if there's a value, gdal can open it.

Reply:
Yep, my thought exactly.  I've added an inline comment to clarify this assumption: 4a9400b7